app/views/admin/organizations/index.html.erb

```diff
@@ -13,10 +19,14 @@
     <div class="row">
       <div class="col-10">
         <h5 class="mb-1">
-          <% if policy([:manage, organization]).edit? %>
+          <% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
```

Comment:
There might be a bug here. Doesn't this bit mean once we enable the flipper flag, _everyone_ will see a link?

Reply:
These checks are actually unnecessary, the organizations are already scoped by the controller, and the controller actions have authorization checks. Not sure why we put them in in the first place tbh.

Reply:
Oh I see. If you can get to this page in the first place, then you can see everything. Cool, that works for me 👍